src/test/scala/is/hail/methods/ExprSuite.scala

```diff
@@ -585,6 +588,56 @@ class ExprSuite extends SparkSuite {
     // FIXME: parser should accept minimum Long/Int literals
     // assert(eval[Long](Long.MinValue.toString+"L").contains(Long.MinValue))
     // assert(eval[Long](Long.MinValue.toString+"l").contains(Long.MinValue))
+
+    {
+      val x = eval[Map[String,IndexedSeq[Int]]]("[1,2,3,4,5].groupBy(k => if (k % 2 == 0) \"even\" else \"odd\")")
+      assert(x.isDefined)
+      x.get.keySet should contain theSameElementsAs Seq("even", "odd")
+      x.get("even") should contain theSameElementsAs Seq(2,4)
+      x.get("odd") should contain theSameElementsAs Seq(1,3,5)
```

Comment:
I want to actually write this as:

```scala
(eval[Map[String,IndexedSeq[Int]]]("[1,2,3,4,5].groupBy(k => if (k % 2 == 0) \"even\" else \"odd\")")
  should contain (contain theSameElementsAs ("even" -> (contain theSameElementsAs Seq(2,4)),
                                             "odd" -> (contain theSameElementsAs Seq(1,3,5)))))
```

or really:

```scala
(eval[Map[String,IndexedSeq[Int]]]("[1,2,3,4,5].groupBy(k => if (k % 2 == 0) \"even\" else \"odd\")")
  shouldBe Some(Map("even" -> exactlyInAnyOrder(2,4), "odd" -> exactlyInAnyOrder(1,3,5))))
```

But scala test matchers are not compassable at all.